core/src/OC/dialogs.js

```diff
@@ -534,9 +534,8 @@ const Dialogs = {
 
 			self.$filePicker.ocdialog({
 				closeOnEscape: true,
-				// max-width of 600
-				width: 600,
-				height: 500,
+				width: '70vw',
+				height: '60vh',
```

Comment:
Should have a max-width and max-height as well but not sure what values would be best, @jancborchardt @skjnldsv?

Reply:
and min too :+1:

Reply:
I am not super good in css, should I set the following:

```suggestion
				mix-width: '70vw',
				max-width: '70vw',
				min-height: '60vh',
				max-height: '60vh',
```
or

```suggestion
				mix-width: '600',
				width: '70vw',
				min-height: '500',
				height: '60vh',
```
?